src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbOperations.java

```diff
@@ -38,11 +38,14 @@
 
     void deleteCollection(String collectionName);
 
-    <T> List<T> delete(DocumentQuery query, Class<T> entityClass, String collectionName);
+    <T> List<T> delete(DocumentQuery query, String collectionName, Class<T> entityClass,
```

Comment:
I think these public should have some comments. For example, it's unclear here why I need `partitionKeyNames` for deletion

Reply:
Yes, we will add comments for this interface in another PR. As the legacy code has no description here.